forc-test/src/ecal.rs

```diff
@@ -31,6 +31,9 @@ impl Syscall {
             Syscall::Fflush { fd } => {
                 let mut f = unsafe { std::fs::File::from_raw_fd(*fd as i32) };
                 let _ = f.flush();
+
+                // Dont close the fd
+                std::mem::forget(f);
```

Comment:
Opening a lot of these fd would introduce a mem-leaks - but im assuming we are fine with that?

Reply:
'from_raw_fd' does not open the 'fd', nor does it allocates at the heap.
Do you mean other kind of leakeage?

Reply:
just reading about `std::mem::forget` from my understanding the destructure (or `drop` trait) is not run for `f` - preventing cleanup of `f`.
This would imply this variable `f` will remain in memory for the duration of the process.